featuros-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,23 @@
+project(':featuros-gradle-plugin') {
+    apply plugin: 'groovy'
+    apply plugin: 'maven-publish'
+
+    dependencies {
+        compile gradleApi()
+        compile localGroovy()
+
+        testCompile 'org.spockframework:spock-core:1.0-groovy-2.3'
```

Comment:
There are problems with test execution when using 2.4, will investigate it further, as explicitly defining groovy version doesn't help - there is still 2.3 version coming from somewhere.